modules/phase_field/test/tests/DeformedGrain/DeformedGrain.i

```diff
@@ -40,6 +40,7 @@
     type = PolycrystalVoronoi
     rand_seed = 81
     coloring_algorithm = bt
+    int_width = 0
```

Comment:
that's the default, why add it?

Reply:
int_width is in `[GlobalParams]` with a different value

Reply:
why?

Reply:
I don't know. I looked at the file again and `int_width` is only needed by one other block. I will switch it to specify in that block only.